### PR TITLE
Update instructions for error handling in bash

### PR DESCRIPTION
**Pull request by Philosoft**
The only way to complete this exercise right now is to read tests. PR adds bare bones instructions on what is required

Fixes: #564 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by IsaacG** [comment]
> The only way to complete this exercise right now is to read tests.

By and large this is true across multiple tracks and by design, leaning pretty hard into the test-driven development philosophy.

This file is synced directly from the [upstream problem spec](https://github.com/exercism/problem-specifications/blob/main/exercises/error-handling/description.md). **If** we opted to flesh out the instructions, it would need to be done via an amend file.

**Comment by glennj** [comment]
Philosophically speaking, as exercism promotes TDD, getting students to read the tests is exactly what we should be doing. However, I agree that this exercise is a bit under-documented.

As [documented](https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md), the place to add this track-specific material is in the `exercises/practice/error-handling/.docs/instructions.append.md` file.

As per exercism's [markdown style guide](https://github.com/exercism/docs/blob/main/building/markdown/markdown.md), all markdown files start with a level-1 header. In this case, the process that joins instructions.md and instructions.append.md drop the header from the append file, so just add a placeholder header.

Regarding the text specifically, I'd prefer the instructions to be a bit less explicit

> The goal of this exercise is to consider the _number_ of arguments passed to your program. If there is exactly one argument, print a greeting message. Otherwise print an error message.

**Comment by Philosoft** [comment]
Ouch. I am sorry. Apparently, I missed mentioning of TDD somewhere. Then maybe there is no need for this change at all 🤔

**Comment by glennj** [comment]
I think this is still a worthy PR, just needs some changes.

**Comment by Philosoft** [comment]
@glennj used your text and moved it to `exercises/practice/error-handling/.docs/instructions.append.md` file

File: exercises/practice/error-handling/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Specific instructions
```
Comment (glennj) [inline exercises/practice/error-handling/.docs/instructions.append.md:1]: The process that generates README (that concatenates instructions.md and instructions.append.md) strips out level-1 headers.

Please start the file with:
```markdown
# Dummy header

## Bash-specific instructions
```
All of the instructions.append.md files in the bash track are similarly incorrect: I'll create an issue to address that.

**Review by glennj — CHANGES_REQUESTED** [review]

File: exercises/practice/error-handling/.docs/instructions.append.md (hunk 1 above)
Comment (Philosoft) [reply exercises/practice/error-handling/.docs/instructions.append.md:1]: I wasn't sure that the real heading was needed, but 'Dummy header' seemed out of place 😅

Comment (Philosoft) [reply exercises/practice/error-handling/.docs/instructions.append.md]: https://github.com/exercism/bash/pull/576/commits/0d523b4e5871249198cf82e7aa517b5de46c6f21 ✅

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
This is what the instructions look like now:
image

**Comment by glennj** [comment]
Oops, a typo: `Bash-pecific`. we'll need another PR to fix that.

**Pull request merged**